neo4j-jdbc-bolt/src/main/java/org/neo4j/jdbc/bolt/BoltNeo4jResultSet.java

```diff
@@ -290,6 +291,10 @@ private Value fetchPropertyValue(String key, String property) throws SQLExceptio
 	}
 
 	private Value fetchValueFromLabel(String label) throws SQLException {
+		if (this.current == null) {
```

Comment:
what does this change fix?

Reply:
the test with `UNWIND [] AS nope RETURN nope`